hypervisor/src/mshv/mod.rs

```diff
@@ -562,6 +562,18 @@ impl cpu::Vcpu for MshvVcpu {
                     debug!("Exception Info {:?}", { info.exception_vector });
                     Ok(cpu::VmExit::Ignore)
                 }
+                hv_message_type_HVMSG_X64_APIC_EOI => {
```

Comment:
Anyway we get an exit here? I think we are not.

Reply:
During our testing we encountered this exit here while passing through a PCI device to guest, disabling MSIX support.

Reply:
I think we can only get this if we enable legacy PCI INTx support. It uses level-triggered interrupt, which requires explicitly acking the vector.

Reply:
We have PCI INTx support for VFIO and vfio-user devices.